src/main/views/webContentEvents.ts

```diff
@@ -121,6 +122,20 @@ export class WebContentsEventManager {
                 return {action: 'deny'};
             }
 
+            // Check for valid URL
+            // Let the browser handle invalid URIs
```

Comment:
This comment does not seem accurate, if I understood the goal of the PR correctly.